incuna-sass/components/_forms.sass

```diff
@@ -95,7 +95,7 @@ label
             @content
 
 @include field-selectors
-    @extend %inline-block
+    display: inline-block
```

Comment:
Should we keep the default behaviour by including `vertical-align: middle` with all these?